src/ResourceManager/Resources/AzureRM.Resources.psd1

```diff
@@ -126,7 +126,7 @@ CmdletsToExport = 'Get-AzureRmProviderOperation', 'Remove-AzureRmRoleAssignment'
 # VariablesToExport = @()
 
 # Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
-AliasesToExport = @()
+AliasesToExport = 'Get-AzureRmResourceProviderAction'
```

Comment:
Please add to changelog.md and this is good.

Reply:
Done.